plugins/stack-overflow/src/home/StackOverflowQuestions/Content.tsx

```diff
@@ -24,6 +24,7 @@ import {
   ListItemSecondaryAction,
   ListItemIcon,
 } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
```

Comment:
Any particular reason not to import `Typography` on line 19?

Reply:
Afaik this is the preferred way to import stuff, instead of relying on tree shaking MUI recommends this way of importing components, many examples in the docs import with full path, can't remember the reason though...